rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -505,7 +505,7 @@ http {
     server {
         # Use the port {{ $all.ListenPorts.Status }} (random value just to avoid known ports) as default port for nginx.
         # Changing this value requires a change in:
-        # https://github.com/kubernetes/ingress-nginx/blob/master/controllers/nginx/pkg/cmd/controller/nginx.go
+        # https://github.com/kubernetes/ingress-nginx/blob/master/internal/ingress/controller/nginx.go
```

Comment:
I wonder whether the comment is still relevant. Does not look like relevant anymore at least to me.

Reply:
I recently discovered Ingress and think it's cool, but am also new here and making my through the source for the first time. Apologies if this PR doesn't add value - my hope was contribute something that I found personally helpful/interesting, even if just a tiny URL swap.

If the proper change here is to just delete the stale comment, by all means let's. 
Perhaps not a bad idea to go through the entire file and clean up no longer relevant stuff while we're at it?

Reply:
@onpaws thanks fro your contribution. The whole comment does not add much value, let's delete it. If you're curious though that value comes from https://github.com/kubernetes/ingress-nginx/blob/master/cmd/nginx/flags.go#L152.

> Perhaps not a bad idea to go through the entire file and clean up no longer relevant stuff while we're at it?

feel free to do it, but I'd rather do this kind of cleanup case-by-case when I notice.